examples/ERC721TokenReceiver/openzeppelin-contracts-4.7.3/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
I feel like I must have been confusing before... DO NOT COMMIT openzeppelin into our repository in any way, whether as the raw files or as the tarball

Reply:
Ok. I was going off of the Jira ticket description of using a "local copy of openzeppelin

Reply:
I'm sorry for being vague.